tests/test_annotation.py

```diff
@@ -289,6 +304,43 @@ def test_default_multicategory_gt_upload(dataset):
     )
 
 
+def test_scene_category_gt_upload(dataset):
+    annotation = SceneCategoryAnnotation.from_json(
+        TEST_SCENE_CATEGORY_ANNOTATIONS[0]
+    )
+    response = dataset.annotate(annotations=[annotation])
+
+    assert response["dataset_id"] == dataset.id
+    assert response["annotations_processed"] == 1
+    assert response["annotations_ignored"] == 0
+
+
+def test_default_scene_category_gt_upload(dataset):
+    annotation = SceneCategoryAnnotation.from_json(
+        TEST_DEFAULT_SCENE_CATEGORY_ANNOTATIONS[0]
+    )
+    response = dataset.annotate(annotations=[annotation])
+
+    assert response["dataset_id"] == dataset.id
+    assert response["annotations_processed"] == 1
+    assert response["annotations_ignored"] == 0
+
+
+def test_non_existent_taxonomy_scene_category_gt_upload(dataset):
+    annotation = SceneCategoryAnnotation.from_json(
+        TEST_NONEXISTENT_TAXONOMY_SCENE_CATEGORY_ANNOTATIONS[0]
+    )
+    response = dataset.annotate(annotations=[annotation])
+
+    assert response["dataset_id"] == dataset.id
+    assert response["annotations_processed"] == 0
+    assert response["annotations_ignored"] == 0
+    assert (
+        f'Taxonomy {TEST_NONEXISTENT_TAXONOMY_SCENE_CATEGORY_ANNOTATIONS[0]["taxonomy_name"]} does not exist in dataset {dataset.id}'
+        in response["errors"][0]
+    )
```

Comment:
This is too specific of a check. We've been burned by tests breaking when somebody changes an error message. 

If you care about some attribute in the error (e.g. ref_id, dataset_id ...) is always shown in the SDK, check for inclusion of those fields. Otherwise feel free to drop the error check.